src/components/application_manager/include/application_manager/resumption/resumption_data_json.h

```diff
@@ -175,32 +181,32 @@ class ResumptionDataJson : public ResumptionData {
    */
   void Persist() OVERRIDE;
 
-  resumption::LastState& last_state() const {
```

Comment:
deprecate, don't remove

Reply:
@jacobkeeler Done in 051714ae2f3af31a9104199aa3c6f3e8937807a1